packages/gradle/src/plugin/nodes.ts

```diff
@@ -44,7 +44,8 @@ export interface GradlePluginOptions {
   testTargetName?: string;
   classesTargetName?: string;
   buildTargetName?: string;
-  [taskTargetName: string]: string | undefined;
+  excludeTestFiles?: string[];
```

Comment:
Remove this option. Change the glob so that it specifically finds `Test.java`, `Tests.java`, `Test.kt`, and `Tests.kt` files.